v2.0/releases/index.md

```diff
@@ -8,16 +8,19 @@ Use the version selector at the top-right of this page to view a different relea
 
 ## v2.0.0-beta
 
-| Component              | Version |
-|------------------------|---------|
-| felix                  | 2.0.0-beta.3 |
-| calicoctl              | v1.0.0-beta |
-| calico/node            | v1.0.0-beta |
-| calico/cni             | v1.5.0 |
-| libcalico              | v0.18.0 |
-| calico-bird            | v0.1.0  |
-| libnetwork-plugin      | v1.0.0-beta |
-| calico/kube-policy-controller | v0.5.0 |
+| Component                     | Version                                                                                    |
+|-------------------------------+--------------------------------------------------------------------------------------------|
```

Comment:
I think the table columns got messed up a bit.  You're missing a `|`.

I prefer to not make the separators stretch the full width of the table (makes it harder to read and isn't necessary for it to format properly).

You should be able to leave the column headers unchanged in this PR.